doc/developer_guide.md

```diff
@@ -74,9 +76,29 @@ go test -p 1 -short ./...
 
 Please note that also `-short` tests need `-p 1` because extension integration tests share a directory for building a test extension. Tests will fail randomly without `-p 1`.
 
-## Static Code Analysis
+#### Manual Integration Tests With SaaS
```

Comment:
This repository is public, right? I'm thinking how much sense this makes outside our organization or if we should have it as internal docs

Reply:
You are right. But this does not reveal any secrets. Having separate internal docs will make it only confusing for developers.